packages/config/config/devices/0x019b/4512689.json

```diff
@@ -21,9 +21,25 @@
-		"exclusion": "1. Put the Z-Wave gateway or control panel into Exclusion mode (please refer to the Z-Wave gateway or control panel manual). \n2. Within 1.5 seconds, press the Function button 3 times and the Smoke Detector will be removed from the Z-Wave network",
-		"reset": "(Only use factory reset when network Control Panel/Gateway is missing or inoperable)\n1. Remove the batteries of the Smoke Detector first.  \n2. Press and hold the Function button. While holding the Function button, power on the Smoke Detector by re-inserting the batteries, wait for 20 seconds to factory reset.",
-		"manual": "https://products.z-wavealliance.org/ProductManual/File?folder=&filename=product_documents/3933/HeatitZ-SmokeDetector230V.pdf"
+		"inclusion": "Please refer to your primary controller manual on how to enter add/remove mode. The device may only be added or removed from the network if the primary controller is in add/remove mode, and may now be added to the system via the primary controller",
+		"exclusion": "Please refer to your primary controller manual on how to enter add/remove mode. The device may only be added or removed from the network if the primary controller is in add/remove mode, and may now be added to the system via the primary controller",
+		"reset": "After the calibration process is finished, you may access the add/remove mode by pressing the function button three times within 1.5 seconds. Removing the device will reset it to factory defaults",
+		"manual": "https://media.heatit.com/709"
+	},
+
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"description": "Lifeline. (Normally used by the Z-WaveController) Sends Battery Report, Notification Report, Sensor Multilevel Report, Device Reset Locally Notification",
+			"maxNodes": 1,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Sensor Basic Set",
+			"description": "Basic set. When the smoke detector is active, it will send basic set (0xFF).When the smoke detector is deactived, it will send basic set (0x00)",
```

Comment:
```suggestion
			"description": "When the smoke detector is active, it will send Basic Set (0xFF) to this group. When the smoke detector is deactived, it will send Basic Set (0x00).",
```